### ISSUE TITLE
ground truth labels for VGGFace and Age

### ISSUE THREAD

**Issue by MohammadSamragh** [PurduePAML/TrojanNN#7]
Thanks for the great work! 

Is there a way to extract the ground-truth labels for the training data images? I am specifically interested in the VGGFace and Age recognition datasets.
